examples/AuthServer.js

```diff
@@ -0,0 +1,51 @@
+const express = require('express');
+const uport = require('../lib/index.js')
+const bodyParser = require('body-parser')
+
+const app = express()
+app.use(bodyParser.json({ type: '*/*' }))
+const signer = uport.SimpleSigner('955d253bca62bf380e34a36ed3802895e4adc297fa861e22bb7ee3e66714592b')
+
+const authCredentials = new uport.AuthCredentials({
+  appName: 'Credential Tutorial',
+  address: '0x15bcc15904f2317365cdb6beaf473b2315405763',
+  signer: signer
+})
+
+app.get('/authrequest', (req, res) => {
+    // Browser requests to start authentication and receives a request token in response
+    credentials.createRequest({
+      request: ['name'],
+      // A reachable ip address
+      callbackUrl: 'http://192.168.1.101:8081/authResponse'
+    }).then( (requestToken) => {
+      const uri = 'me.uport:me?requestToken=' + requestToken
+      const.log(uri)
+      res.send(uri);
+    })
+})
+
+app.post('/authresponse', (req, res) => {
+  // Mobile app posts response to challenge here
+  const jwt = req.body.access_token
+  const response = 'Successful Authentication'
+  receive(jwt, response).then( credentials => {
+    console.log(response)
+  }).catch(err => {
+    console.log('Authentication Failed')
+  })
+})
+
+app.get('/authresponse/:pairID', (req, res) => {
```

Comment:
@pelle yeah I am okay with sending it back the browser. Sending the response back to the server would be stronger model, since it minimizes (almost entirely) the chance that it is leaked. The token (and/or additional information) could be leaked. For example on mobile when passing from app to app (uport to browser, maybe another app captures it), or capturing in browser history since in the url. But yes this risk seems minimal right now and passing it to the browser would help keep the same flows were already have in mobile and in the libraries.